src/common/components/TimeAgo/TimeAgo.ts

```diff
@@ -1,15 +1,4 @@
-import React from 'react'
-import cn from 'classnames'
-
-export type TTimeAgoProps = {
-  date: number
-  className?: string
-}
-
-export default function TimeAgo({
-  date,
-  className,
-}: TTimeAgoProps): JSX.Element {
+export default function timeAgo(date: number): string {
```

Comment:
`common/components` is only for components, as `timeAgo` is an utility function now, move it to `common/utilities/format.ts` - `format` seems to be close by the meaning.

Reply:
Updated